scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts

```diff
@@ -100,7 +102,7 @@ export const flashLoanPools = {
   },
   arbitrum: {
     usdc: {
-      tokenAddress: addresses.arbitrum.USDT,
+      tokenAddress: addresses.arbitrum.USDC, // USDC/USDC_E/.01% pool
```

Comment:
There's more USDC to borrow in the `USDC/USDC_E` pool (~$2M) than in the `USDT/USDC_E` pool (~$750K)